CDCgov/prime-reportstream
# Bump Updated Gradle wrapper to 7.4.2

This PR updates the gradle version to 7.4.2.

Test Steps:
1. Build the project locally by running `./gradlew clean package` (you MUST use gradlew) and verifying the build is successful.
2. Run the RS services locally using gradle by running `./gradlew run` and verify RS runs as expected.
3. Verify this PR's build passes.

## Changes
- update the gradle version to 7.4.2.

## Checklist

### Testing

### Process

## Linked Issues
- N/A

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
